tuf/client/client.go

```diff
@@ -111,10 +110,6 @@ func (c Client) checkRoot() error {
 		return fmt.Errorf("Cached root sha256 did not match snapshot root sha256")
```

Comment:
Just FYI, as far as we're concerned hash collisions are impossible. The only reason to have the size checks is to protect against infinite data attacks.